### PR TITLE
Heatmap_Imag_Txt Rvw

### PR DESCRIPTION
**Pull request by AnaSasso**

### CODE REVIEW

File: docs/toolbox/accessibility_indicators/heatmaps/gravity.md [hunk 1]
```diff
@@ -9,33 +9,34 @@ import thematicIcon from "/img/toolbox/data_management/join/toolbox.webp";
...
+![Gravity-based Heatmap in GOAT](/img/toolbox/accessibility_indicators/heatmaps/gravity_based/heatmap_gravity_based.webp "Gravity-based Heatmap in GOAT")
   
 :::info 
-The calculation of heatmaps is currently possible in specific regions. Upon selecting a routing mode, you will see a **geofence** on the map indicating supported regions.
+The Heatmaps are available for specific regions. 
+Upon selecting a `Routing Mode` the **geofence** for heatmap will be displayed on the map, indicating supported regions.
 
-TODO: Update this image to reflect the latest geofence for heatmaps
 <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
   <img src={require('/img/toolbox/accessibility_indicators/catchments/geofence.png').default} alt="Geofence for catchment area calculation in GOAT" style={{ maxHeight: "400px", maxWidth: "400px", alignItems:'center'}}/>
```
Comment (nihar1024) [inline docs/toolbox/accessibility_indicators/heatmaps/gravity.md:35]: I think this still needs to be updated?

Comment (nihar1024) [reply docs/toolbox/accessibility_indicators/heatmaps/gravity.md:35]: Heatmaps currently support only Germany. You can maybe use the same image of the public transport catchment area geofence.

File: docs/toolbox/accessibility_indicators/heatmaps/gravity.md [hunk 2]
```diff
@@ -237,7 +238,7 @@ Depending on your configuration, the calculation might take a few minutes. The [
 
 TODO: Update result image
 
-![Catchment Area Calculation Result in GOAT](/img/toolbox/accessibility_indicators/catchments/catchment_result.png "Catchment Area Calculation Result in GOAT")
+![Heatmap Gravity-Based Calculation Result in GOAT](/img/toolbox/accessibility_indicators/heatmaps/gravity_based/heatmap_gravity_result.webp "Heatmap Gravity-Based Calculation Result in GOAT")
```
Comment (nihar1024) [inline docs/toolbox/accessibility_indicators/heatmaps/gravity.md:241]: This image looks nice!
Small change: could we avoid highlighting the accessibility value? This value is not easy to understand and so we would rather encourage users to observe the color gradient and visual aspects of the result.

File: docs/toolbox/accessibility_indicators/heatmaps/gravity.md [hunk 3]
```diff
@@ -9,33 +9,34 @@ import thematicIcon from "/img/toolbox/data_management/join/toolbox.webp";
...
 
-**Gravity-based Heatmaps** use a color-coded map to visualize the accessibility of [POIs](../../further_reading/glossary#point-of-interest-poi "What is a POI?") and point-based data.
+**Gravity-based Heatmaps** uses a color-coded map to visualize the accessibility of a points cluster, eg.: [POIs](../../further_reading/glossary#point-of-interest-poi "What is a POI?"). 
 
 ## 1. Explanation
 
-Visualized as a color-coded hexagonal grid, heatmaps take into account real-world transport and street networks to compute accessibility. When you specify a routing mode (such as `walking`, `bicycle`, etc.) and a maximum travel duration, a grid reflecting all areas accessible under these conditions is produced.
+Visualized as a color-coded hexagonal grid, heatmaps takes into account real-world transport and streets networks, to compute accessibility. Specify a routing mode (`Walk` or `Bicycle`), select `Opportunity Layer`. Set a maximum travel duration. The result shall display the color-coded hexagonal grid for all the areas accessible under those conditions. The color scale referes to the local accessibility.
+
+:::info INFO
```
Comment (nihar1024) [inline docs/toolbox/accessibility_indicators/heatmaps/gravity.md:18]: Minor, but a one-line gap needs to be left below the opening `:::info` and one line above the closing `:::`

File: docs/toolbox/accessibility_indicators/heatmaps/gravity.md (hunk 1 above)
Comment (AnaSasso) [reply docs/toolbox/accessibility_indicators/heatmaps/gravity.md:35]: For some reason it doesn't show a Germany only geofence for me. If you could provide de image.

**Pull request merged**